From 3fa9db42f3b34f97929662a461c9a07982505178 Mon Sep 17 00:00:00 2001
From: Denys Dmytriyenko <denys@ti.com>
Date: Tue, 25 Aug 2015 10:05:15 -0400
Subject: [PATCH] configure: bump path length from 256 to 512 characters

Increase the path length that gets hardcoded into generated config.cpp file
from 256 to 512 characters, as nativesdk path can be quite long.

Also update length of EXT_PREFIX and HOST_PREFIX now.

Change-Id: If98dd57160efe9c98c36148cdf872f50b3d38118
Signed-off-by: Denys Dmytriyenko <denys@ti.com>
---
 configure.pri | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure.pri b/configure.pri
index 47882afeb6..a50aa0e176 100644
--- a/configure.pri
+++ b/configure.pri
@@ -870,10 +870,10 @@ defineTest(qtConfOutput_preparePaths) {
 
     $${currentConfig}.output.qconfigSource = \
         "/* Installation Info */" \
-        "static const char qt_configure_prefix_path_str  [12+256] = \"qt_prfxpath=$$config.input.prefix\";" \
+        "static const char qt_configure_prefix_path_str  [12+512] = \"qt_prfxpath=$$config.input.prefix\";" \
         "$${LITERAL_HASH}ifdef QT_BUILD_QMAKE" \
-        "static const char qt_configure_ext_prefix_path_str   [12+256] = \"qt_epfxpath=$$config.input.extprefix\";" \
-        "static const char qt_configure_host_prefix_path_str  [12+256] = \"qt_hpfxpath=$$config.input.hostprefix\";" \
+        "static const char qt_configure_ext_prefix_path_str   [12+512] = \"qt_epfxpath=$$config.input.extprefix\";" \
+        "static const char qt_configure_host_prefix_path_str  [12+512] = \"qt_hpfxpath=$$config.input.hostprefix\";" \
         "$${LITERAL_HASH}endif" \
         "" \
         "static const short qt_configure_str_offsets[] = {" \
